src/App.js

```diff
@@ -1,25 +1,75 @@
-import logo from './logo.svg';
+import React, { useEffect, useState } from "react";
+import Recipe from './Recipe';
 import './App.css';
 
-function App() {
-  return (
+const App = () => {
+  //API authentication things from EDAMAM
+  const APP_ID = "d1b01914";
+  const APP_KEY = "02b7f93c25e936d6f2ba18462d3cc9ed";
+
+
+  const [recipes, setRecipes] = useState([]);
+  const [search, setSearch] = useState("");
+  //want to only fetch data when search button is clicked
+  const [query, setQuery] = useState('chicken'); //default
+
+  useEffect(() => {
+    getRecipes();
+    console.log('use effect running');
+  }, [query]); 
+  // adding [] as a second arg makes sure effect is only run once
+  // can also add state in this arg, only when counter changes this useEffect runs
```

Comment:
Nice takeaways here! Looks like you understand the basic idea of useEffect and useState (: